releasenotes/notes/service-entry-instances-store-memory-leak.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+releaseNotes:
+- |
+  **Fixed** service entry instances store memory leak
```

Comment:
We donot need this as this bug is not in previous releases, so no need to expose out.